include/TMS/dev/TCA9545A.h

```diff
@@ -25,15 +25,17 @@ enum TCA9545A_BUS {
  * TCA9545A I2C Multiplexer Driver
  * https://www.ti.com/lit/ds/symlink/tca9545a.pdf
  */
-class TCA9545A : I2CDevice::I2CDevice {
+class TCA9545A {
 public:
 
     /**
      * Constructor for the TCA9545A driver
      *
      * @param i2c I2C device for communicating with TCA
```

Comment:
These should be labeled `@param[in]` to indicate that they're inputs. This should be done for all the method documentation